frontend/src/app/general/nav-bar-menu/nav-bar-menu.component.html

```diff
@@ -0,0 +1,79 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+<mat-list>
+  <a mat-list-item (click)="navBarService.toggle()" routerLink="/projects"
+    >Projects</a
+  >
+  <mat-divider></mat-divider>
+  <a mat-list-item (click)="navBarService.toggle()" routerLink="">Sessions</a>
+  <mat-divider></mat-divider>
+  <a
+    mat-list-item
+    (click)="navBarService.toggle()"
+    *ngIf="userService.user?.role === 'administrator'"
+    [routerLink]="['/sessions', 'overview']"
+    >Session overview</a
```

Comment:
In the top menu bar this is called "Overview".

Reply:
Good point, I'll rename "Overview" to "Session overview".

Reply:
Migrated to data structure, is now consistent.